src/overview/pricing.md

```diff
@@ -77,12 +77,12 @@ Production plans come in three flavors:
 
 ### Multiple Applications in a single project
 
-All Platform.sh plans support multiple applications in a single cluster but
-they all share the global resource of the cluster.
+All Platform.sh plans support multiple applications in a single cluster, but
+they share the global resources of the cluster.
```

Comment:
Single-line it, and I dispute this comma.

Reply:
It's two independent clauses separated by a coordinating conjunction. There is not a single style guide that would call that comma incorrect.

Reply:
Stet.